utils/lib/pb_type_xml.py

```diff
@@ -100,6 +100,19 @@ def add_switchblock_locations(xml):
     })
 
 
+def get_site_pin_wire(equivalent_site_pin, site_pins):
+    no_prefix_equivalent_site_pin_wire = equivalent_site_pin.wire.split('_'
```

Comment:
This function needs a docstring.

Reply:
@acomodi Can you help me with the docstring there?

Reply:
@mkurc-ant This change went in the commit with the flake8 linter fixes, the docstrings can be found here: https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1484/commits/2e4d5a332b208a55510bdcac8d0b5d7e4aabf439 and here is also a fix for the assertion: https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1484/commits/357add1d840d92666b8a373c3856cc03bf03efc2

Reply:
Thx